packages/reporters/liquidation-reporter/liquidation-reporter.ts

```diff
@@ -166,6 +155,17 @@ export async function fetchUmaEcosystemData() {
         if (samplePriceFeed.invertPrice) invertedPrice = true;
         if (samplePriceFeed.priceFeeds && samplePriceFeed.priceFeeds[0].invertPrice) invertedPrice = true;
 
+        // If the collateral type is stable, then we must invert the price feed by default. These kinds of feeds (such
+        // as the BasketSpreadPriceFeed) do not contain the `invertPrice` flag, but they are all "inverted". As additional
+        // stable collaterals are added this will need to be updated or the page will show up no liquidatable positions
+        // for that collateral type.
+        if (
```

Comment:
this was a tricky one. all price feeds for stable collaterals are technically "inverted", such as `BasketSpreadPriceFeed`. Without this no stable coin liquidations show up correctly.